fe/fe-core/src/test/resources/sql/tpchcost/q22.sql

```diff
@@ -43,34 +43,24 @@ PARTITION: UNPARTITIONED
 
 RESULT SINK
 
-17:MERGING-EXCHANGE
+16:MERGING-EXCHANGE
 
 PLAN FRAGMENT 1
 OUTPUT EXPRS:
-PARTITION: HASH_PARTITIONED: 32: substring
+PARTITION: UNPARTITIONED
 
 STREAM DATA SINK
-EXCHANGE ID: 17
+EXCHANGE ID: 16
 UNPARTITIONED
 
-16:SORT
+15:SORT
 |  order by: <slot 32> 32: substring ASC
 |  offset: 0
 |
-15:AGGREGATE (update finalize)
+14:AGGREGATE (update finalize)
 |  output: count(*), sum(6: C_ACCTBAL)
 |  group by: 32: substring
 |
-14:EXCHANGE
-
-PLAN FRAGMENT 2
-OUTPUT EXPRS:
-PARTITION: UNPARTITIONED
-
-STREAM DATA SINK
-EXCHANGE ID: 14
-HASH_PARTITIONED: 32: substring
```

Comment:
Why not shuffle by substring?

Reply:
Because this child(avg without group by) is support gather property, all data are gather to one instance.
In actually, I think the plan maybe isn't best, shuffle work on mutli-node maybe more fast than gather on one-node